controllers/sessionController.js

```diff
@@ -0,0 +1,58 @@
+const User = require("../models/User");
+const parseVErr = require("../util/parseValidationErr");
+const csrf = require('host-csrf');
+
+const registerShow = (req, res) => {
+    res.render("register");
+};
+
+const registerDo = async (req, res, next) => {
+    let validation_errors = false
+    if (req.body.password != req.body.password1) {
+        req.flash("error", "The passwords entered do not match.");
+        validation_errors = true;
+    }
+    try {
+        await User.create(req.body);
+    } catch (e) {
+        if (e.constructor.name === "ValidationError") {
+            parse_v(e, req);
```

Comment:
Need to use the same name as  the const you define at top:
            parseVErr(e, req);